src/scss/layout/_header.scss

```diff
@@ -62,3 +62,18 @@
     grid-area: 1 / 1 / span 1 / -1;
   }
 }
+
+[data-header~='stacked'] {
```

Comment:
I like this approach. I think great to use it on the home page as well. I'm not sure if we have stacked headers in other places. @SondraE might know.